### PR TITLE
Collapsible team description

### PR DESCRIPTION
**Pull request by hikes101**
Right now you can click on a team member and the description appears below but also moves the other images around. I wanted to fix that and have the clicked on member move to the top. However, I couldn't get it to work so I want to close this branch for now since there are other tasks with higher priority.

### CODE REVIEW

**Comment by hikes101** [comment]
[Port About page](https://trello.com/c/TpxRJfDE/15-port-about-page)

File: src/routes/home.js [hunk 1]
```diff
@@ -20,6 +20,19 @@ function prepareHome(data) {
...
+  result.calendar.forEach(element => {
+    const weekday = moment(element.date, "DD.MM.YYYY")
+      .format("dddd")
+      .toLowerCase();
+    element.dayicon = `img/calendar/${weekday}.png`;
+  });
+
+  return result;
+}
+
```
Comment (pheeria) [inline src/routes/home.js]: This here seems to be false. Please, consult @starmaerker :)

Comment (pheeria) [reply src/routes/home.js]: I think it should not be there at all.

File: views/about.hbs [hunk 2]
```diff
@@ -13,14 +13,19 @@
...
-      <div class="team-box">
+      <div class="team-box" data-toggle="collapse" data-target="#{{id}}" onclick="(e) => {console.log(e);}">
         <img src={{img}} alt={{name}} class="img-fluid rounded-circle" style="height: 160px; width: 160px;">
         <p class="lead">{{name}}</p>
       </div>
+      <div id="{{id}}" class="collapse" data-parent="#team">
+        <div class="card-body team-description">
+          {{description}}
+        </div>
+      </div>
```
Comment (pheeria) [inline views/about.hbs]: This seems like a good candidate for a partial and a snapshot test

File: views/about.hbs [hunk 3]
```diff
@@ -13,14 +13,19 @@
   </div>
 </div>
 <div class="container-fluid text-center">
-  <div class="team-row text-center">
-    <div class="col-sm-10">
+  <div class="team-row">
+    <div class="col-sm-10" id="team">
       {{#each team}}
-      <div class="team-box">
+      <div class="team-box" data-toggle="collapse" data-target="#{{id}}" onclick="(e) => {console.log(e);}">
```
Comment (pheeria) [inline views/about.hbs]: @jarjan can you please review this? My Bootstrap knowledge is very basic :)

File: src/routes/home.js (hunk 1 above)
Comment (hikes101) [reply src/routes/home.js]: @pheeria @starmaerker Interesting...I guess this happened when I used 'git pull origin master'. I now copied home.js from the current master and replaced the file. Thanks for noticing!

File: views/about.hbs (hunk 2 above)
Comment (hikes101) [reply views/about.hbs]: @pheeria I will do this in a new branch once this branch is merged. I already tagged myself in the trello card and moved it to 'active'.

File: views/about.hbs (hunk 3 above)
Comment (jarjan) [reply views/about.hbs]: All good, except onclick is unnecessary. 
https://getbootstrap.com/docs/3.4/javascript/#collapse

Comment (hikes101) [reply views/about.hbs]: Thanks, I deleted it. I used onclick for debugging and forgot to remove it afterwards.

**Pull request merged**